### PR TITLE
Periodic_3_triangulation_3 demo: Initialize Point_3

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

A default  constructed point is not necessarily at the origin.   Fix warning in [testsuite ](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.14-I-114/Periodic_3_triangulation_3_Demo/TestReport_lrineau_ArchLinux-CXX17-Release.gz)

## Release Management

* Affected package(s): Periodic_3_triangulation_3

### CODE REVIEW

File: Periodic_3_triangulation_3/demo/Periodic_3_triangulation_3/Scene.cpp [hunk 1]
```diff
@@ -1413,7 +1413,7 @@ void Scene::gl_draw_location() {
                 Point p = tet_to_draw.vertex((i+1)&3);
                 Point q = tet_to_draw.vertex((i+2)&3);
                 Point r = tet_to_draw.vertex((i+3)&3);
-                Vector c= (Vector(Point(),p)+Vector(Point(),q)+Vector(Point(),r))/3.;
+                Vector c= (Vector(Point(0,0,0),p)+Vector(Point(0,0,0),q)+Vector(Point(0,0,0),r))/3.;
                 Point cp = Point(c.x(),c.y(),c.z());
```
Comment (lrineau) [inline Periodic_3_triangulation_3/demo/Periodic_3_triangulation_3/Scene.cpp]: I think a better code would be:
```c++
Point cp = CGAL::centroid(p, q, r);
```

https://doc.cgal.org/latest/Kernel_23/group__centroid__grp.html

https://doc.cgal.org/latest/Kernel_23/classKernel_1_1ConstructCentroid__3.html

Comment (afabri) [reply Periodic_3_triangulation_3/demo/Periodic_3_triangulation_3/Scene.cpp]: Just did it. I was blind by only thinking about fixing the "uninitialized" warning.

**Comment by afabri** [comment]
@maxGimeno please add it to the testsuite

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-4.14-Ic-122.shtml

**Pull request merged**